### PR TITLE
Add folder and file for exercise 11: Factorails

### PR DESCRIPTION
**Pull request by ian-deans**

### CODE REVIEW

File: spec_11/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+const findFactorial = ( n ) => {
```
Comment (jrobcodes) [inline spec_11/index.js]: You may omit parentheses for single parameter functions.

File: spec_11/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  let factorial = 1
+  for ( let i = 1; i <= n; i++ ) {
```
Comment (jrobcodes) [inline spec_11/index.js]: Consider adding whitespace for clarity.  I try to divide functions into the setup, body, and return sections with the use of whitespace.

File: spec_11/index.js (hunk 1 above)
Comment (ian-deans) [reply spec_11/index.js]: there needs to be a better face-palm emoji

File: spec_11/index.js (hunk 2 above)
Comment (ian-deans) [reply spec_11/index.js]: Another face-palm moment lol, the essay analogy

File: spec_11/index.js (hunk 1 above)
Comment (jrobcodes) [reply spec_11/index.js]: Eh, it's a matter of style more than anything else <shrug>

File: spec_11/index.js (hunk 2 above)
Comment (jrobcodes) [reply spec_11/index.js]: I need to capture and publish that somewhere.  :)

Comment (ian-deans) [reply spec_11/index.js]: I completely agree, even though its been a long time, I feel essay format is so entrenched in my brain it just makes sense to use that as a way to organize code as well.

File: spec_11/index.js (hunk 1 above)
Comment (ian-deans) [reply spec_11/index.js]: Psh, whats life without a little style?

**Pull request merged**